modules/webbrowser/properties/propertywidgetcef.cpp

```diff
@@ -33,6 +33,42 @@
 
 namespace inviwo {
 
+class CefDOMSearchId : public CefStringVisitor {
+public:
+    CefDOMSearchId(const std::string& htmlId, PropertyWidgetCEF* widget,
+                   const CefRefPtr<CefFrame> frame)
+        : CefStringVisitor(), htmlId_(htmlId), widget_(widget), frame_(frame){};
+
+    void Visit(const CefString& string) OVERRIDE {
+        domString_ = string;
+
+        // Remove any occurences of " or ' from the domString_ to remove possible variations on html
+        // id declarations.
+        domString_.erase(std::remove(domString_.begin(), domString_.end(), '"'), domString_.end());
```

Comment:
How does the search cope with commented code?

Reply:
If someone would comment "id=buttonId" or such, the visitor will act as if that is an actual html element.

Reply:
Would either fix it or add it as a limitation in the documentation

Reply:
Added a loop to remove any html comments from the DOM-string in this commit: https://github.com/inviwo/inviwo/pull/275/commits/3b37bda616c4204362f9a5ceed5d19a2742a60c6